.github/workflows/create-linux-wheel.yml

```diff
@@ -0,0 +1,44 @@
+name: Build Linux Wheel
```

Comment:
This workflow is a temporary measure to build a wheel for testing and should be removed before merge.